scratch/test.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import print_function
+import numpy as np
+from numpy import histogram
+
+np.seterr(all='raise')
+# tiny_shift = 1e-8
+tiny_shift = 0.0
+count, x_loc = np.histogram(
+    np.array([1.0], np.float32),
+    bins=1,
+    range=np.array([1.0 + tiny_shift, 2.0]).astype('f8')
+)
+
+print('done')
```

Comment:
This file should probably be removed

Reply:
yes, this was just a temporary test (probably i committed it by mistake)